server/sonar-server/src/main/java/org/sonar/server/issue/ws/ChangelogAction.java

```diff
@@ -61,7 +61,9 @@ public ChangelogAction(DbClient dbClient, IssueFinder issueFinder) {
   @Override
   public void define(WebService.NewController context) {
     WebService.NewAction action = context.createAction(ACTION_CHANGELOG)
-      .setDescription("Display changelog of an issue")
+      .setDescription("Display changelog of an issue<br/>." +
+        "Require the 'Browse' permission on the project of the specified issue<br/>." +
```

Comment:
Requires the following permission:
- 'Browse' permission on the project of the specified issue